addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -2124,3 +2129,38 @@ def test_invoice_sent_to_additional_partner(self):
         self.assertTrue(follower_mail)
         self.assertNotIn('access_token=', follower_mail.body_html,
                       "The followers should not bet sent the access token by default")
+
+    def test_purchase_uom_on_vendor_bills(self):
+        def create_move_and_return_uom(move_type, product):
```

Comment:
No need to recreate a helper for creating invoices, we could just use `self.invoice` or another existing helper (`init_invoice` maybe ?)